pkg/operation/seed/seed.go

```diff
@@ -511,6 +567,33 @@ func BootstrapCluster(k8sGardenClient kubernetes.Interface, seed *Seed, config *
 		return err
 	}
 
+	var (
+		grafanaTLSOvderride   = grafanaTLS
+		prometheusTLSOverride = prometheusTLS
+		kibanaTLSOverride     = kibanaTLS
+	)
+
+	wildcardCert, err := GetWildcardCertificate(context.TODO(), k8sSeedClient.Client())
+	if err != nil {
+		return err
+	}
+
+	if wildcardCert != nil {
+		grafanaTLSOvderride = wildcardCert.GetName()
+		prometheusTLSOverride = wildcardCert.GetName()
+		kibanaTLSOverride = wildcardCert.GetName()
+	}
+
+	wildcardCerts := &corev1.SecretList{}
```

Comment:
I assume the lines 587-595 are a left-over and can be removed in favor of line 576?

Reply:
Nice catch, thanks